test/cbmc/proofs/MQTTAgent_ResumeSession/Makefile

```diff
@@ -45,6 +45,6 @@ PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/agent_command_pool_stubs.c
 PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/agent_message_stubs.c
 PROOF_SOURCES += $(SRCDIR)/test/cbmc/stubs/core_mqtt_stubs.c
 
-PROJECT_SOURCES += $(SRCDIR)/source/mqtt_agent.c
```

Comment:
Lines 34-36 needs to be updated from __CPROVER_file_local_mqtt_agent_c to __CPROVER_file_local_core_mqtt_agent_c